packages/hardhat-core/src/internal/hardhat-network/provider/provider.ts

```diff
@@ -284,6 +284,7 @@ export class HardhatNetworkProvider extends EventEmitter
       this._reset.bind(this),
       (loggingEnabled: boolean) => {
         this._loggingEnabled = loggingEnabled;
```

Comment:
Feels like this variable can be removed. But I'm fine with doing it later.

Reply:
Probably right. But let's keep this small for now.